show/main.py

```diff
@@ -1187,7 +1187,11 @@ def protocol(verbose):
     ip.add_command(bgp)
     from .bgp_frr_v6 import bgp
     ipv6.add_command(bgp)
-
+elif device_info.is_supervisor():
```

Comment:
you don't need to add supervisor for bgp setup. supervisor in chassis will not have bgps

Reply:
nvm, you are adding this cmd on sup to execute for all linecards